src/react-components/src/__tests__/components/Breadcrumbs-test.tsx

```diff
@@ -4,22 +4,31 @@ import * as Enzyme from "enzyme";
 import * as React from "react";
 import * as Mocha from "mocha";
 
-import Breadcrumbs from "../../components/02-molecules/menus/Breadcrumb/Breadcrumb";
+import Breadcrumbs from "../../components/02-molecules/menus/Breadcrumbs/Breadcrumbs";
 
 describe("Breadcrumbs", () => {
   let wrapper: Enzyme.ShallowWrapper<{}, {}>;
   let shortBreadcrumb = [{ url: "#", text: "test1" }, { url: "#", text: "test2" }];
+  let components = [
+    <a role="link" href="#test1"><span>Hello</span></a>, <a role="link" href="#test2">Goodbye</a>
+  ];
 
   beforeEach(() => {
     wrapper = Enzyme.shallow(<Breadcrumbs breadcrumbs={shortBreadcrumb} />);
   });
 
   it("Renders a tag with custom text", () => {
+    wrapper = Enzyme.shallow(<Breadcrumbs breadcrumbs={shortBreadcrumb} />);
     let links = wrapper.render();
 
-    expect(links.find("a")).to.have.lengthOf(2);
-    expect(links.find("a").first().text()).to.equal("test1");
-    expect(links.find("a").last().text()).to.equal("test2");
+    expect(links.find("a.breadcrumbs__link")).to.have.lengthOf(2);
+    expect(links.find("a.breadcrumbs__link").first().text()).to.equal("test1");
```

Comment:
I would expect these to resolve to "Hello" and "Goodbye", am I just not understanding what `.first().text()` equates to with Enzyme?

Reply:
It's because this guy is using `shortBreadcrumb` instead of `components` (I should change these names, to make it clear what it's testing, but see L21)

Reply:
Ahhh, I see